testing-guidance/workflow-tests/o3de-project-manager/create-a-gem/create-a-gem-workflow-tests.md

```diff
@@ -5,18 +5,102 @@ These workflows center around testing the basic functionality of the Project Man
+| Platform Support - Platforms Unspecified | <ol><li>Create a Gem, without including any desired [Target Platforms](#target-platforms)</li></ol>                                                                                                                                                                                                                                                                                      | <ul><li>Created Gem Shows no entry related to Platforms in its description within the **Gem Catalog**.</li></ul>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+
+
+### Workflow: Add a remote gem from O3DE Project Manager
+
+**Workflow Docs**
+[O3DE Gem Repositories](https://www.o3de.org/docs/user-guide/gems/repositories/overview/)
+
+**Repository Configuration**
+
+A git repository must contain a `repo.json` at its root which points to templates. The `repo.json` contains a gems 
+list which points to the remote gems within said repository.
+
+See [sample-code-gems repo.json](https://github.com/o3de/sample-code-gems/blob/main/repo.json) as an example.
+
+```json
+{
+    "gems": [
+        "https://raw.githubusercontent.com/o3de/sample-code-gems/main/atom_gems/AtomTutorials",
+        "https://raw.githubusercontent.com/o3de/sample-code-gems/main/cpp_gems/ShapeExample",
+        "https://raw.githubusercontent.com/o3de/sample-code-gems/main/py_gems/PyShapeExample"
+    ]
+}
+```
+
+**Workflow Requirements**
+* A built project to add a remote gem to (a blank template project is acceptable).
+* Locally installed tools for the remote source being used (git installed with credentials for GitHub, for example)
+* A remote Git repository (GitHub, AWS CodeCommit, BitBucket, self-hosted, ETC) that has a remote project.
+  * Examples: 
+    * sample-code-gems (https://github.com/o3de/sample-code-gems.git)
```

Comment:
I think we should add the canonical repo too `https://canonical.o3de.org` ([repo.json](https://canonical.o3de.org/repo.json))

Reply:
I'm not super familiar with remote gems but I like the idea of providing a list of links to some that are commonly used and known to be stable.

Reply:
Added